### PR TITLE
Added one 6DoF object tracking paper, one hand tracking paper.

### PR DESCRIPTION
**Pull request by HLrwth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2365,7 +2368,9 @@ Neurocomputing, 547, 2023. [Code](https://github.com/miracleyoo/yelan-dynamic-dv
 - <a name="Naeini20sensors"></a>Rudnev, V., Golyanik, V., Wang, J., Seidel, H.-P., Mueller, F., Elgharib, M., Theobalt, C.,  
 *[EventHands: Real-Time Neural 3D Hand Reconstruction from an Event Stream](https://arxiv.org/pdf/2012.06475)*,  
 arXiv, 2020. [Project page](https://gvv.mpi-inf.mpg.de/projects/EventHands/)
-
+- <a name="xue2022_evnonrigid"></a>Xue, Y., Li, H., Leutenegger, S., Stueckler, J.,
```
Comment (guillermogb) [inline README.md:2371]: key: Xue22bmvc

File: README.md [hunk 2]
```diff
@@ -477,7 +477,10 @@ The Visual Computer, 2021. [PDF](https://link.springer.com/article/10.1007/s0037
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [Project](https://zhangjiqing.com/publication/iccv21_fe108_tracking/), [PDF](https://arxiv.org/abs/2109.09052), [code](https://github.com/Jee-King/ICCV2021_Event_Frame_Tracking), [dataset](https://zhangjiqing.com/dataset/).
 - <a name="Dietsche21iros"></a>Dietsche, A., Cioffi, G., Hidalgo-Carrio, J., Scaramuzza, D.,  
 *[Powerline Tracking with Event Cameras](http://dx.doi.org/10.1109/IROS51168.2021.9636824)*,  
-IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [PDF](https://rpg.ifi.uzh.ch/docs/IROS21_Dietsche.pdf), [dataset](https://download.ifi.uzh.ch/rpg/powerline_tracking_dataset/), [YouTube](https://www.youtube.com/watch?v=KnBJqed5qDI), [code](https://github.com/uzh-rpg/line_tracking_with_event_cameras).  
+IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [PDF](https://rpg.ifi.uzh.ch/docs/IROS21_Dietsche.pdf), [dataset](https://download.ifi.uzh.ch/rpg/powerline_tracking_dataset/), [YouTube](https://www.youtube.com/watch?v=KnBJqed5qDI), [code](https://github.com/uzh-rpg/line_tracking_with_event_cameras).
+- <a name="li2021_evtracking"></a>Li, H., Stueckler, J.,
```
Comment (guillermogb) [inline README.md:481]: key: Li21icra

Comment (guillermogb) [reply README.md:481]: For these papers, are there PDFs available without a paid subscription? (e.g., arXiv)

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (HLrwth) [reply README.md]: Updated the keys and link. Thanks!

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -478,8 +478,8 @@ IEEE Int. Conf. Computer Vision (ICCV), 2021. [Project](https://zhangjiqing.com/
 - <a name="Dietsche21iros"></a>Dietsche, A., Cioffi, G., Hidalgo-Carrio, J., Scaramuzza, D.,  
 *[Powerline Tracking with Event Cameras](http://dx.doi.org/10.1109/IROS51168.2021.9636824)*,  
 IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [PDF](https://rpg.ifi.uzh.ch/docs/IROS21_Dietsche.pdf), [dataset](https://download.ifi.uzh.ch/rpg/powerline_tracking_dataset/), [YouTube](https://www.youtube.com/watch?v=KnBJqed5qDI), [code](https://github.com/uzh-rpg/line_tracking_with_event_cameras).
-- <a name="li2021_evtracking"></a>Li, H., Stueckler, J.,
-*[Tracking 6-DoF Object Motion from Events and Frames](https://doi.org/10.1109/ICRA48506.2021.9561760)*,  
+- <a name="Li21icra"></a>Li, H., Stueckler, J.,
+*[Tracking 6-DoF Object Motion from Events and Frames](https://arxiv.org/pdf/2103.15568.pdf)*,
```
[the remark this answers is not in the document]
Comment (HLrwth) [reply README.md:482]: Sorry about that, I added the blank spaces after author names.

**Review by guillermogb — APPROVED** [review]
thanks

**Pull request merged**